Src/CSharpier/SyntaxPrinter/SyntaxNodePrinters/BaseMethodDeclaration.cs

```diff
@@ -173,18 +199,31 @@ public static Doc Print(CSharpSyntaxNode node)
             if (constraintClauses != null)
             {
                 docs.Add(
-                    groupId != null
-                        ? ConstraintClauses.PrintWithConditionalSpace(constraintClauses, groupId)
+                    parameterGroupId != null
+                        ? ConstraintClauses.PrintWithConditionalSpace(
+                                constraintClauses,
+                                parameterGroupId
+                            )
```

Comment:
Do you think we should half-indent ternary operators?
That way, the closing parenthesis will be aligned with the start of the method.

And things that aren't method calls (e.g. binary operators) usually indent themselves anyway, so they should look fine.